vectors/cryptography_vectors/pkcs7/enveloped-rsa-oaep.pem

```diff
@@ -0,0 +1,16 @@
+-----BEGIN PKCS7-----
```

Comment:
If you wouldn't mind a separate PR with this vector, we can get it merged ASAP.

Reply:
#11998